MapboxNavigation/InstructionsCardContainerView.swift

```diff
@@ -129,18 +129,17 @@ public class InstructionsCardContainerView: StylableView {
         setGradientLayer(for: lanesView)
         setGradientLayer(for: nextBannerView)
         
-        layer.cornerRadius = style.cornerRadius
+        layer.cornerRadius = 20.0
```

Comment:
This property should still be overridable through the new mechanism. We use it in Apex. The mobile (iPhone) UI uses a different corner radius than the in-car (iPad) UI.